mage_ai/server/scheduler_manager.py

```diff
@@ -24,9 +32,20 @@ def run_scheduler():
             sentry_dsn,
             traces_sample_rate=SENTRY_TRACES_SAMPLE_RATE,
         )
+    disable_new_relic = False
```

Comment:
Can we try reducing duplicate code?
You can add a newrelic subfolder in https://github.com/mage-ai/mage-ai/tree/master/mage_ai/services and put newrelic related methods in it so that you can reuse it.

Reply:
Yes, created a common function under `services/newrelic` and reused them in different scripts.